application/common/core.cpp

```diff
@@ -372,30 +375,56 @@ static int sdlPriorityEvents(void* userdata, SDL_Event* event)
     return true;
 }
 
-enum  optionIndex { UNKNOWN, HELP, FROM_RUBY };
+enum  optionIndex { UNKNOWN, VERSION, HELP, FROM_RUBY, APP_TYPE, LOOP };
+enum  optionType { DISABLE, ENABLE, OTHER };
```

Comment:
To be sure - I mean the enum name (optionType) since that can be used as a type. The actual enum item names are fine as is.